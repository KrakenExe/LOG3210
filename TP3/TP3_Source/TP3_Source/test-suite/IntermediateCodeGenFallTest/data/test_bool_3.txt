Declaration
bool a;
bool b;
EndDeclaration

a = true && false;
b = true && true;