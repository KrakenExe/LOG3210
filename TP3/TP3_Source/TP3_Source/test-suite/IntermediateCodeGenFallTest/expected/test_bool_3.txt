goto _L2
a = 1
goto _L1
_L2
a = 0
_L1
b = 1
goto _L0
_L3
b = 0
_L0
